integration-test/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "integration-test",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "print": "vite-node src/script/print.ts"
```

Comment:
This is just utility. It'll make `out.json` at the same directory of given input file, which is the result of our parser.

Run following scripts at the root of the repository:

```
npm --workspace=integration-test run print <your_dxf_file>
```